client.go

```diff
@@ -29,14 +31,14 @@ func NewClient(masterAPIKey string, baseURL string) *Client {
 }
 
 // Get the feature state associated with the environment for a given feature
-func (c *Client) GetEnvironmentFeatureState(environmentAPIKey string, featureName string) (*FeatureState, error) {
-	url := fmt.Sprintf("%s/environments/%s/featurestates/", c.baseURL, environmentAPIKey)
+func (c *Client) GetEnvironmentFeatureState(environmentKey string, featureID int64) (*FeatureState, error) {
```

Comment:
Will this be a breaking change again?

Reply:
Yep

Reply:
Ok, all good as long as we call it out in the docs.